indexing-service/src/test/java/io/druid/indexing/common/task/SameIntervalMergeTaskTest.java

```diff
@@ -68,14 +69,13 @@
   private final CountDownLatch publishCountDown = new CountDownLatch(1);
   private final IndexSpec indexSpec;
   private final ObjectMapper jsonMapper;
-  private IndexIO indexIO;
 
   public SameIntervalMergeTaskTest()
   {
+    EmittingLogger.registerEmitter(new NoopServiceEmitter());
```

Comment:
yes, otherwise the test will fail.
```
  public AlertBuilder makeAlert(Throwable t, String message, Object... objects)
  {
    if (emitter == null) {
      final String errorMessage = String.format(
          "Emitter not initialized!  Cannot alert.  Please make sure to call %s.registerEmitter()", this.getClass()
      );
```
the emitter is registered in EmittingLogger.registerEmitter